contrib/test_jenkins.sh

```diff
@@ -72,6 +72,10 @@ MAKE="make"
 MAKEP="make -j${parallel_jobs}"
 export AUTOMAKE_JOBS=$parallel_jobs
 
+#
+# Override maven repository path, to cache the downloaded packages accross tests
+#
+export maven_repo=${WORKSPACE}/.deps
```

Comment:
Is `${WORSPACE}` per job folder? Would it preserve between jobs?

Reply:
it's per job, the problem was that even for same jenkins job .deps were removed every time we did "make clean"
(i think that per-node it's too risky because of multiple usages in same time)